project/users/views.py

```diff
@@ -1,57 +1,101 @@
- #project/users/views.py
+# project/users/views.py
 
-#imports
-from flask import Flask,request,jsonify,abort,make_response,Blueprint
+# imports
+import datetime
+import jwt
+import psycopg2
+from flask import Flask, request, jsonify, abort, make_response, Blueprint
+from project.config import conn, Config
 
 
-#configure blueprint
-users = Blueprint('users',__name__,template_folder='templates')
 
-users_list = []
-session = []
-logged_in = ""
-requests =[]
 
-#test if routes are working
-@users.route('/')
-def request():
-        return "hello" 
+# configure blueprint
+users = Blueprint('users', __name__, template_folder='templates')
 
-@users.errorhandler(404)
-def request_not_found(error):
-        return make_response(jsonify({'error':'Not found'}),404)  
+cur = conn.cursor()
 
+def jwt_auth_encode(userid):
+    # to generate the auth token
 
+    try:
+        payload = {
+            'exp': datetime.datetime.utcnow() + datetime.timedelta(days=0, seconds=10),
+            'iat': datetime.datetime.utcnow(),
+            'sub': userid
+        }
+        return jwt.encode(
+            payload,
+            Config.SECRET
+        )
+    except Exception as e:
+        raise e
 
-@users.route('/api/v1/users/', methods=['POST'])
-def create_user():
-   
-        
-    app_request = {
 
-                        'id': len(users_list)+1,
-                        'email': request.json['email'],
-                        'password': request.json['password'],
 
-            
-                        }
-    users_list.append(app_request)
-    return jsonify({'app_request':app_request}),201     
+def decode_auth_token(auth_token):
+    # decode auth token
+    try:
+        payload = jwt.decode(auth_token,Config.SECRET)
+        return payload['sub']
+    except Exception as e:
+        return("Try logging in again", e)
 
-@users.route('/api/v1/users/login', methods=['POST'])
-def login_user():
+@users.route('/')
+def index():
+    return "hello"
+
+@users.errorhandler(404)
+def request_not_found(error):
+    return make_response(jsonify({'error': 'Not found'}), 404)
 
-    email = request.json['email']
-    password = request.json['password']
 
-    for u in users_list:
-        if u['email'] == email and u['password'] == password:
-            global logged_in
-            logged_in = u['email']
-            return jsonify({'logged_in': True}),200
-    return jsonify({'logged_in': False}),400 
+@users.route('/auth/signup', methods=['POST'])
+def create_user():
+    form = request.get_json()
+    print('form', form)
```

Comment:
There is no need for you to add print statements in your work if they're not adding any value to it.